big_o_exercise/bigo.js

```diff
@@ -0,0 +1,57 @@
+/*
+1. O(n)
+2. O(log(n))
+3. O(1)
+4. O(n^3)
+5. O(n)
+6. O(log(n))
+7. O(n * log(n))
+8. O(2^n)
+9. O(1)
+10. O(n2)
+*/
+
+//1 O(n)
```

Comment:
not sure whether these are time or space complexities. make sure you record both.